lib/plugins/print/print.js

```diff
@@ -96,6 +97,14 @@ class Print {
     // preparation for printing the service for the user.
     return getServerlessConfigFile(process.cwd())
       .then((svc) => {
+        const isRunInService = !!svc;
```

Comment:
is this not redundant?  the risk of the file being deleted between plugin load and reloading the file here seems small.

Reply:
It is indeed! I must have implemented it here early on before changing strategy. It has been removed 👍

Reply:
Your comment here had led me to expect this code would disappear: https://github.com/serverless/serverless/pull/5298/files#diff-5c58e3ad980e17d98380999420228cfdR100

For the record, just noting I was a little confused.  Thanks for the changes!
Also for the record, I'm not a maintainer so I am limited in my ability to move things forward here, sorry :D

Reply:
😬  Sorry my commit failed to push due to connectivity issues - should be sorted now.

No worries thank you for your review 👍